action/protocol/staking/vote_bucket.go

```diff
@@ -87,18 +87,18 @@ func (vb *VoteBucket) fromProto(pb *stakingpb.Bucket) error {
 		return err
 	}
 
-	createTime, err := ptypes.Timestamp(pb.GetCreateTime())
-	if err != nil {
+	if err := pb.GetCreateTime().CheckValid(); err != nil {
 		return err
 	}
-	stakeTime, err := ptypes.Timestamp(pb.GetStakeStartTime())
-	if err != nil {
+	createTime := pb.GetCreateTime().AsTime()
+	if err := pb.GetStakeStartTime().CheckValid(); err != nil {
 		return err
 	}
-	unstakeTime, err := ptypes.Timestamp(pb.GetUnstakeStartTime())
-	if err != nil {
+	stakeTime := pb.GetStakeStartTime().AsTime()
+	if err := pb.GetUnstakeStartTime().CheckValid(); err != nil {
 		return err
 	}
+	unstakeTime := pb.GetUnstakeStartTime().AsTime()
```

Comment:
did not call `CheckValid` here, see above

Reply:
I've called it here
https://github.com/iotexproject/iotex-core/blob/6bf4d714ec4e8ed0811b612f58e220d44acd9328/action/protocol/staking/vote_bucket.go#L98-L101